superset-frontend/src/dashboard/containers/DashboardPage.tsx

```diff
@@ -161,12 +164,18 @@ const DashboardPage: FC = () => {
   useEffect(() => {
     // eslint-disable-next-line consistent-return
     async function getDataMaskApplied() {
+      const permalinkStateKey = getUrlParam(URL_PARAMS.permalinkStateKey);
       const nativeFilterKeyValue = getUrlParam(URL_PARAMS.nativeFiltersKey);
       let dataMaskFromUrl = nativeFilterKeyValue || {};
 
       const isOldRison = getUrlParam(URL_PARAMS.nativeFilters);
-      // check if key from key_value api and get datamask
-      if (nativeFilterKeyValue) {
+      // check if key from temporary_cache api and get datamask
```

Comment:
I think this comment belongs to the `else` statement?

Reply:
Removing, as the functions are pretty self explanatory